### PR TITLE
create-chiselstrike-app: Generate README.md

### PR DESCRIPTION
**Pull request by penberg**
Suggested by @ricardodalarme.

### CODE REVIEW

File: packages/create-chiselstrike-app/template/README-template.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+## Getting Started
+
+To get going, run:
+
+```console
+npm run dev
+```
+
+which starts a local development server of the example code.
```
Comment (ricardodalarme) [inline packages/create-chiselstrike-app/template/README-template.md:13]:
```suggestion
Which starts a local development server of the example code.
```

File: packages/create-chiselstrike-app/template/README-template.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# ChiselStrike Application
```
Comment (ricardodalarme) [inline packages/create-chiselstrike-app/template/README-template.md:1]: Can this title be the project name as flutter does?

Comment (glommer) [reply packages/create-chiselstrike-app/template/README-template.md]: @penberg I pushed a commit that does this.

**Review by glommer — APPROVED** [review]

File: packages/create-chiselstrike-app/template/README-template.md (hunk 2 above)
Comment (penberg) [reply packages/create-chiselstrike-app/template/README-template.md]: 👏

**Pull request merged**